rest/replicatortest/replicator_test.go

```diff
@@ -8681,6 +8681,9 @@ func TestReplicationConfigUpdatedAt(t *testing.T) {
 	currTime := configResponse.UpdatedAt
 	createdAtTime := configResponse.CreatedAt
 
+	// avoid flake where update at seems to be the same (possibly running to fast)
+	time.Sleep(500 * time.Nanosecond)
+
 	resp = activeRT.SendAdminRequest("PUT", "/{{.db}}/_replicationStatus/replication1?action=stop", "")
 	rest.RequireStatus(t, resp, http.StatusOK)
 
```

Comment:
Down below where we do `assert.Greater` we can use `AssertTimeGreaterThan` - it has built in handling for this particular flavour of Windows-specific failure to tolerate equal times even if the sleep does fail.

If we wrote a bug around this where we never update timestamp, it'd show up on other OSes

Reply:
Since this is only on windows that it fails IIRC, maybe we should not sleep, because I don't know if it will help? But I agree about this timestamp thing.